docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -299,7 +299,28 @@ extensions to alter behavior based on these settings. Settings should be well do
 ClusterClass authors can understand usage and expected behaviour.
 
 Settings can be provided for individual external patches by providing them in the ClusterClass `.spec.patches[*].external.settings`.
-This can be used to overwrite settings at the ExtensionConfig level for that patch.
+This can be used to overwrite settings at the ExtensionConfig level for that patch. Below is an example of `ExtensionConfig` -
```

Comment:
```suggestion
This can be used to overwrite settings at the ExtensionConfig level for that patch. 


Below is an example of `ExtensionConfig` -
```

Reply:
Yes you are right, I will add those section in the settings.